### ISSUE TITLE
Host binding with hostname (instead TCP port)

### ISSUE THREAD

**Issue by nevergone** [axllent/mailpit#213]
Mailhog allow port binding with hostname (for ex. `smtp:25`), but mailpit not. Related code snippet (from config.go):

```
        re := regexp.MustCompile(`^((\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})|(\[([\da-fA-F:])+\])):\d+$`)
        if !re.MatchString(SMTPListen) {
                return errors.New("SMTP bind should be in the format of <ip>:<port>")
        }
        if !re.MatchString(HTTPListen) {
                return errors.New("HTTP bind should be in the format of <ip>:<port>")
        }
```

**Comment by axllent** [comment]
Thanks for the feedback. I have made a change that will allow this, and will be included in the next release.

**Comment by axllent** [comment]
This has been released in v1.10.2. Please confirm this works as expected?

**Comment by nevergone** [comment]
Thanks, it's working! :)
